Source/OxyPlot/Rendering/OxyCircle.cs

```diff
@@ -0,0 +1,197 @@
+        public bool Contains(double x, double y)
+        {
+            return this.Center.DistanceTo(new ScreenPoint(x, y)) < this.Radius;
+        }
+
+        /// <summary>
+        /// Determines whether the specified point is inside the circle.
+        /// </summary>
+        /// <param name="p">The point.</param>
+        /// <returns><c>true</c> if the circle contains the specified point; otherwise, <c>false</c>.</returns>
+        public bool Contains(ScreenPoint p)
+        {
+            return this.Contains(p.x, p.y);
+        }
+
+        /// <summary>
+        /// Determines whether this instance and another specified <see cref="T:OxyCircle" /> object have the same value.
+        /// </summary>
+        /// <param name="other">The circle to compare to this instance.</param>
+        /// <returns><c>true</c> if the value of the <paramref name="other" /> parameter is the same as the value of this instance; otherwise, <c>false</c>.</returns>
+        public bool Equals(OxyCircle other)
+        {
+            return this.Center.Equals(other.Center) && this.Radius.Equals(other.Radius);
+        }
+
+        /// <summary>
+        /// Returns a <see cref="System.String" /> that represents this instance.
+        /// </summary>
+        /// <returns>A <see cref="System.String" /> that represents this instance.</returns>
+        public override string ToString()
+        {
+            return string.Format(CultureInfo.InvariantCulture, "({0}, {1}, {2})", this.Center.X, this.Center.Y, this.Radius);
+        }
+
+        /// <summary>
+        /// Returns a <see cref="System.String" /> that represents this instance.
+        /// </summary>
+        /// <param name="format">The format.</param>
+        /// <param name="formatProvider">The format provider.</param>
+        /// <returns>
+        /// A <see cref="System.String" /> that represents this instance.
+        /// </returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            const string Separator = ", ";
+            var builder = new StringBuilder();
+            builder.Append("(");
+            builder.Append(this.Center.X.ToString(format, formatProvider));
+            builder.Append(Separator);
+            builder.Append(this.Center.Y.ToString(format, formatProvider));
+            builder.Append(Separator);
+            builder.Append(this.Radius.ToString(format, formatProvider));
+            builder.Append(")");
+            return builder.ToString();
+        }
+
+        /// <summary>
+        /// Returns a circle that is expanded or shrunk by the specified radius amounts.
+        /// </summary>
+        /// <param name="dr">The amount by which to expand or shrink the radius of the circle.</param>
+        /// <returns>The expanded/shrunk <see cref="OxyCircle" />.</returns>
+        public OxyCircle Inflate(double dr)
```

Comment:
Not used; would omit for the time being as the inflate could be misunderstood as a factor.

Reply:
(Similarly for annulus)

Reply:
as above